pyomo/contrib/edi/__init__.py

```diff
@@ -0,0 +1,34 @@
+#  ___________________________________________________________________________
+#
+#  Pyomo: Python Optimization Modeling Objects
+#  Copyright (c) 2008-2023
+#  National Technology and Engineering Solutions of Sandia, LLC
+#  Under the terms of Contract DE-NA0003525 with National Technology and
+#  Engineering Solutions of Sandia, LLC, the U.S. Government retains certain
+#  rights in this software.
+#
+#  Development of this module was conducted as part of the Institute for
+#  the Design of Advanced Energy Systems (IDAES) with support through the
+#  Simulation-Based Engineering, Crosscutting Research Program within the
+#  U.S. Department of Energy’s Office of Fossil Energy and Carbon Management.
+#
+#  This software is distributed under the 3-clause BSD License.
+#  ___________________________________________________________________________
+
+# Recommended just to build all of the appropriate things
+import pyomo.environ
```

Comment:
This was following an @jsiirola comment that I should always import pyomo.environ, but perhaps I misunderstood the intention of the comment.  I included it here because it is possible to build an EDI model without ever importing pyomo.environ